### ISSUE TITLE
bedtools getfasta fails to give output

### ISSUE THREAD

**Issue by BioinformaticsLabAtMUN** [BioContainers/containers#290]
Hello,

I was working with biocontainers/bedtools and found that getfasta doesn't take input files and it returns always the usage info.

docker pull biocontainers/bedtools
docker run -i -v /Users/lourdes/Documents/Courses/Fall2018/Comp3550_Biol3951/Labs/Lab6_Containers:/data --rm biocontainers/bedtools bedtools getfasta -fi /data/sequenceFromEntrez.fasta -bed /data/domainCoordinates_v2.bed -tab 

Tool:    bedtools getfasta (aka fastaFromBed)
Version: v2.25.0
Summary: Extract DNA sequences into a fasta file based on feature coordinates.

Usage:   bedtools getfasta [OPTIONS] -fi <fasta> -bed <bed/gff/vcf> -fo <fasta> 

Options: 
	-fi	Input FASTA file
	-bed	BED/GFF/VCF file of ranges to extract from -fi
	-fo	Output file (can be FASTA or TAB-delimited)
	-name	Use the name field for the FASTA header
	-split	given BED12 fmt., extract and concatenate the sequencesfrom the BED "blocks" (e.g., exons)
	-tab	Write output in TAB delimited format.
		- Default is FASTA format.

	-s	Force strandedness. If the feature occupies the antisense,
		strand, the sequence will be reverse complemented.
		- By default, strand information is ignored.

	-fullHeader	Use full fasta header.
		- By default, only the word before the first space or tab is used.

However, if I do the same command using another docker container with bedtools, it works:
docker pull genomicpariscentre/bedtools
docker run -i -v /Users/lourdes/Documents/Courses/Fall2018/Comp3550_Biol3951/Labs/Lab6_Containers:/data --rm genomicpariscentre/bedtools bedtools getfasta -fi /data/sequenceFromEntrez.fasta -bed /data/domainCoordinates_v2.bed -tab 
AAN41636.1:146-362	QYQKEFKNSTYSRSSVDMLYSFAKCSGLDLIFGLNALLRTPDLRWNSSNAQLLLDYCSSRGYNISSELGNEPNSFWKKAHILINGSQLGEDFVELHKLLQRSAFQNAKLYGPDIGQPRGKTVKLLRSFLKAGGEVIDSLTWHHYYLNGRIATKEDFLSSDVLDTFILSVQKILKVTKEITPGKKVWLGKTSSAYGAGAPLLSNTFAAGFMWLDKLG
CAF94326.1:113-326	KYRRVKFTETTVDQLHAFANCSGLDLVFGLNALLRTADNRWNSSNARSLLRYCEARRYHMSWELGNEPNSYEKKAGLRLDGRQLGEDFTVLRKILRESRFYRDAGLFGPDVGQPRDHRIDILSGFLQSGAEAVDACTWHHYYLDGREASLEDFLDPDVLDTLREKIGEVLEEVHQVSPGKPVWLGETSSAYGGGAAGLSDTFVAGFMWLDKLG
....

**Comment by osallou** [comment]
Strange, thanks for reporting. @Raynooo can you have a look?

**Comment by Raynooo** [comment]
**biocontainers/bedtools** uses version 2.25.0. Its signature is different from **genomicpariscentre/bedtools**
2.25.0 :    
`bedtools getfasta [OPTIONS] -fi <fasta> -bed <bed/gff/vcf> -fo <fasta>`
Notice how in this case -fo is not an option and needs to be set.
2.27.0 :
`bedtools getfasta [OPTIONS] -fi <fasta> -bed <bed/gff/vcf>`
This time -fo is optional.
    
When building an image using biocontainer/bedtool:2.27.0 recipe, it indeed ran fine with the same arguments as genomicpariscentre/bedtools.

**tl;dr :** 
_docker pull biocontainers/bedtools_ pulls and older version in which _getfasta_'s required arguments were different.

**Comment by Raynooo** [comment]
Now that erroneous _latest_ tags have been removed, it shouldn't be possible to accidentally pull  _bedtools:2.25.0_  instead of a newer one.
